server/hedley/modules/custom/hedley_topic/plugins/content_types/term_title/term_title.inc

```diff
@@ -25,7 +25,10 @@ function hedley_topic_term_title_content_type_render($subtype, $conf, $args, $co
 
   $block = new stdClass();
   $block->module = 'hedley_topic';
-  $block->content = theme('hedley_topic_term_title', ['title' => $wrapper->label()]);
+  $block->content = theme('hedley_topic_term_title', [
+    'title' => filter_xss($wrapper->label()),
+    'description' => $wrapper->description->value(),
```

Comment:
Not using `filter_xss` here because it will remove any style added for the description.